doc/classes/Dictionary.xml

```diff
@@ -109,7 +109,7 @@
 		[/gdscript]
 		[csharp]
 		// This is a valid dictionary.
-		// To access the string "Nested value" below, use `my_dir.sub_dir.sub_key` or `my_dir["sub_dir"]["sub_key"]`.
+		// To access the string "Nested value" below, use `myDict.sub_dir.sub_key` or `myDict["sub_dir"]["sub_key"]`.
```

Comment:
This does not work in C#.

The closest you can get is probably:
```csharp
((Godot.Collections.Dictionary)myDict["sub_dir"])["sub_key"]
```

(or using `dynamic`)

Reply:
I was just fixing the typo with the property name, didn't know that.

Reply:
but hey, I found more places where `dict` was written as `dir`

Reply:
so in the C# example's comments, would the `// Indexing styles can be mixed and matched depending on your needs.` line need to be removed? Since there's only one way?

Reply:
I'd remove it, since it does not apply.

Reply:
Ah, I was worried that it doesn't work like that in C#, but a cursory search made me think it would. I guess the real problem is with multiple levels of access?

Reply:
Yeah, since accessing from a dictionary returns an `object` you need to cast the result to a dictionary to access again (of course, casting something that is not a dictionary to a dictionary will throw an exception).